Dockerfile.ci

```diff
@@ -0,0 +1,18 @@
+ARG commit=latest
+FROM elifesciences/hypothesis-dummy:${commit}
+
+# extract into another image and put here with multi-stage build
```

Comment:
if smoke tests and proofreader go into other images, we can potentially pull them in with a `FROM` and `COPY --from=...` pair. Right now all of this has to be re-executed on each new commit.